src/main/java/com/adobe/cq/testing/junit/rules/Page.java

```diff
@@ -126,40 +123,10 @@ protected String initialTemplatePath() {
      * @throws ClientException if the client cannot be retrieved
      */
     protected CQClient getClient() throws ClientException {
-        return quickstartRule.getAdminClient(CQClient.class);
-    }
-
-    /**
-     * Method to be called before creating the page.
-     * You can override this to perform custom operations before creating the page.
-     * This implementation creates, if needed, the test template and site.
-     *
-     * @throws ClientException if the content cannot be created
-     */
-    protected void prepare() throws ClientException {
```

Comment:
What happened to this? How is the page created now?

Reply:
I've made `Page` more lightweight and remove the template mechanism. What we are currently using is wrong and not technically supported, so we have to come up with a full supported flow (Blueprint> Site> Template> Page). For now, for the scope of what we need with `Page`, this should be enough.